PraceDomowe/Ly_HoangThien_ Homework no.5/Homework no.5 _ Final Version.ipynb

```diff
@@ -0,0 +1,1886 @@
+{
```

Comment:
<p>Correct, -0.1 points for the fact that output is a <code>pd.Series</code>, not a <code>string</code>.</p><p> </p><p><code>0.9 points</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>